iphone/Classes/KrollContext.m

```diff
@@ -659,7 +659,6 @@ - (void)invoke:(KrollContext *)context
     id excm = [KrollObject toID:context value:exception];
     [[TiExceptionHandler defaultExceptionHandler] reportScriptError:[TiUtils scriptErrorValue:excm]];
     pthread_mutex_unlock(&KrollEntryLock);
-    @throw excm;
```

Comment:
why not throw the exception?

Reply:
We already handle the exception on the lines above. Throwing an exception causes the app to crash with an exception that cannot be caught since it's throwing a `KrollObject` and not a valid exception.